src/routes/Player/SubtitlesMenu/SubtitlesMenu.js

```diff
@@ -204,7 +204,15 @@ const SubtitlesMenu = React.memo((props) => {
                         <div className={styles['variants-list']}>
                             {subtitlesTracksForLanguage.map((track, index) => (
                                 <Button key={index} title={track.label} className={classnames(styles['variant-option'], { 'selected': props.selectedSubtitlesTrackId === track.id || props.selectedExtraSubtitlesTrackId === track.id })} data-id={track.id} data-origin={track.origin} data-embedded={track.embedded} onClick={subtitlesTrackOnClick}>
-                                    <div className={styles['variant-label']}>{track.origin}</div>
+                                    <div className={styles['variant-label']}>
+                                        {
+                                            typeof track.label === 'string' && !track.label.startsWith('http') ?
+                                                track.label
+                                                :
+                                                track.lang
+                                        }
+                                        <div className={styles['variant-origin']}>{t(track.origin)}</div>
```

Comment:
In some cases the origin is not fully visible as well. Could you please solve this issue as well?
![image](https://github.com/user-attachments/assets/3d91643a-137a-44dc-8307-47415791293f)